udata/templates/header.html

```diff
@@ -32,70 +32,69 @@
         {% endfor %}
     </ul>
 
-      <ul class="nav navbar-nav navbar-right collapse navbar-collapse">
-        {# <li><a href="{{ url_for('front.metrics') }}">{{ _('Metrics') }}</a></li> #}
-        {% if current_user.is_authenticated() %}
-        <li class="dropdown">
+    <ul class="nav navbar-nav navbar-right collapse navbar-collapse">
```

Comment:
The `div` was broken and I wasn't sure whether the sample dataset i'm preparing were not being displayed for that reason. It was a different reason in the end yet this was fixed.